events-api/src/modules/tickets/dto/payment-ticket.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { IsNotEmpty, IsNumber, IsString, Length, Matches, Validate } from "class-validator";
+import { CardExpiryConstraint } from '../../../shared/validators/cardValidators/card-expiry-date.validator';
+import { CardHolderNameConstraint } from '../../../shared/validators/cardValidators/card-holder-name.validator';
+
+export class PayTicketReqDto {
+  @IsString()
+  @Length(4, 4)
+  @Matches(/^\d{4}$/, { message: 'Card digits must contain only 4 digits' })
+  last4Digits: string;
+
+  @IsString()
+  @Validate(CardExpiryConstraint)
```

Comment:
Create own decorator instead of using Validate

Reply:
Done